FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/ConfigurationValidators/ConfigurationValidatorV1.cs

```diff
@@ -83,6 +83,16 @@ public static void Validate(IServiceCollection services)
                 throw new ConfigurationErrorException($"Target azure container name can not be empty.");
             }
 
+            if (jobConfiguration.MaxConcurrencyCount < 1)
```

Comment:
Do we need to add a const variable for maximum count limitation?
(The value could be set after performance tests.)

Reply:
I think I need to give the default count after performance tests. The maximum count depends on the target machine, and I think we may don't need to limit it.

Reply:
Per our offline discussion, we don't need to limit the maximum count here.